pkg/transform/service.go

```diff
@@ -0,0 +1,11 @@
+package transform
+
+import (
+	"go.expect.digital/translate/pkg/model"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+)
+
+// ProtoServiceFromService converts model.Service to translatev1.Service.
+func ProtoServiceFromService(service *model.Service) *translatev1.Service {
+	return &translatev1.Service{Id: service.ID.String(), Name: service.Name}
```

Comment:
if uuid.UUID is empty (uuid.Nil), `.String()` returns '00000000-0000-0000-0000-000000000000' but we expect empty string value in the proto message. Set only if it's not uuid.Nil.